### ISSUE TITLE
Decide on strategy for item-level content warning

### ISSUE THREAD

**Issue by ggeisler** [sul-dlss/vt-arclight#190]
Separate from any general site content warning, we discussed having a more specific content warning associated with, if I remember correctly, images and video files, though I want to confirm that, and a couple of other questions:

**Q1:** How are we defining which items the item-level content warning applies to? `edm:type: Images` and `edm: Video` (not `edm:type: Sound` or `edm:type: Text`)?

**Q2:** Do we have an idea of what the item-level content warning text should be? Is just a duplicate of the more general content warning statement or something more specific?

**Q3:** How are we assigning the item-level content warning to items? Potential options, though there might be others:
- A: Add to the items' descriptive metadata (which makes it available in other applications, such as Purl)? If so, will we be able to get that out to display in VT ArcLight on a component show page?

- B: Add the content warning to the item's "Use and Reproduction" statement, in which case it is viewable through the media viewer (but initially hidden behind the bottom bar info icon)? But that would only apply to video; images are, I assume displayed in the Mirador viewer, which I don't think surfaces that statement (there is the copyright panel with a Rights section, but I'm not sure if anything gets pulled into that except the Attribution statement and SUL logo)?
- C: Display it directly on the relevant components' show page, using application logic to determine when the component show page is displaying a relevant media item, and if so display the item-level content warning.

I'll create a separate actionable ticket with design, if relevant, after we have answers (Q3 in particular is sort of a blocker to me working on a design).

**Comment by laurensorensen** [comment]
Yes, we only want individual content warnings for Graphic material and Moving Image tagged files (what we are mapping the edm:type terms to, from an ASpace limited list).

Hannah is asking Geoff and Michael now about adding a slate with content warning text to the front of the videos as an item level content warning there. 

It seems like for Graphic material (Images) it won't be possible to display in the viewer without a lot of developer time. 

For me, Option C is preferred but given our limited time frame and the fact that the media player may be changed up in the not too distant future and the fact that Mirador Camille is gone, we may not be able to do it. I'll make it a post standup topic to confirm. 

Option A seems to be the most realistic option. We would have to add with the metadata going to ASpace, if I'm understanding correctly? Where are you imagining it would be presented?

**Comment by ggeisler** [comment]
@laurensorensen Regarding the strategy for displaying a content warning at the item-level for images, perhaps we should discuss this in a meeting. I'm confused by you referring to adding the metadata to ASpace for Option A, because Option A is the one where we add the warning to the item metadata. If we are going to add the warning directly to ASpace, I think that is more like Option C (whether the warning text is in the EAD or the application logic, we're not getting it from the SDR metadata, which is what Option A was suggesting).

At this late stage of the workcycle, I think Option C (whether the warning text is in the EAD or the application logic is something we can discuss with developer input, but if you think it should be part of the EAD a user might download, that would suggest it should go in ASpace) might be the most straightforward. Even if we go with Option C, there's nothing there that precludes you from also adding a content warning to the item SDR metadata, we just might not use it directly for NTA this workcycle (but it could be viewable as part of the the item record in PURL, etc.).

In any case, I'd suggest displaying it something like this, which assumes we can add some conditional code on the component show page that detects when the component is an Image, and if so, displays the content warning, otherwise doesn't display it:

<img width="565" alt="Screen Shot 2022-11-16 at 10 23 26 AM" src="https://user-images.githubusercontent.com/101482/202255728-882c5a77-a73b-41a1-8bb5-08c2914ab8dc.png">

**Comment by thatbudakguy** [comment]
confirmed at 11/16 standup: anything that's an image or video in NTA should get a content warning using option C.

**Comment by laurensorensen** [comment]
RE: Images term change -- @ggeisler I'm sorry about the Images change. I will definitely discuss beforehand with you going forward. We did it because there was duplicative information in the container instance type (a controlled vocab set) in ArchivesSpace, so I mapped all edm:type to the following set so that there wasn't this duplicate set of information (the mapped to is now container instance type). 
Sound = audio (actual text = Audio)
Moving Image = moving_images (display text: Moving Images)
Text = text (display text: Text)
Image = graphic_materials (display text: Graphic Materials

I'm realizing that this doesn't follow our case-pattern (Graphic materials versus Graphic Materials). I am fine to ask developers to change the display set in ArcLight to Images rather than Graphic Materials. Or ask if this can be added to the instance_instance_type limited list in ArchivesSpace. Also if we should set a meeting to discuss, that is ok too.

**Comment by ggeisler** [comment]
@laurensorensen I'm only concerned about how we display things, so no problem with making any changes you need to to better organize the data. But on the display side it is important to make our labels as simple, clear, and consistent as possible for end-users.

Consistency of case: This isn't something I expect you to know, because I'm not sure it is actually documented anywhere, but the SUL style is to use sentence-case in our applications and websites (at least ones the Access team develops). There are exceptions for prominent titles and headings (like application titles or the titles of Spotlight exhibits) and there are UI labels we inherit from other applications that are in title case, but when we can we should use sentence-case so the UI feels consistent to the user. 

Consistency of plurality: I guess this one is more variable, but we use the singular for Resource type in the VT exhibit and for Resource type in Searchworks, so it feels unnecessary and arbitrary to me to use plural in NTA. If we can't control it, sure, not that big of a deal, but if we are making the choice, I'd say we should be consistent with prior practice and use the singular.

And then, yeah, I just don't think "Graphic material" is as clear a term to the end-user as "Image." It might be how an archivist thinks of the content or might be more technically correct but from a display point of view the concern is making clear to the user what it is. "Graphic material" can be interpreted different ways and is not an everyday term like "Image" is.

So if we can't do any of this for whatever reason, fine, but my suggestion is:

- Audio
- Image
- Moving image
- Text

**Comment by laurensorensen** [comment]
Thanks for this Gary! I suppose it also depends on if we're using ArchivesSpace after all, since that graphic_material term is a limited set of data from Spec. I don't know if I can submit a change to that so I'll check first with devs about changing the display only (enable allowing user in AL to change a term display in faceting). 

@marlo-longley or @thatbudakguy Can we easily change the display of one term in a facet? For example, we would like to have:

Audio
Image
Moving image
Text

**Instead of** 

Audio
Graphic Materials
Moving Images
Text

In the list of facets available for container instance type. In ASpace, container instance type is a limited controlled list that is maintained by Spec, so would like to avoid changing it in ASpace if possible.

**Comment by laurensorensen** [comment]
Here is the content warning text: 

Short (for item-level):
Content warning: Imagery and language presented in this archival material may be harmful or traumatizing to some audiences.

Long:
Content warning: Users are advised that material in Taube Archive of the International Military Tribunal (IMT) at Nuremberg, 1945-46 contains language and imagery depicting human rights violations, ethnic cleansing, acts of genocide, wartime violence, and offensive stereotypes of people and cultures. Stanford Libraries makes this material available to facilitate scholarly research and education, and does not endorse the viewpoints within.
